glean-core/csharp/csharp/Glean.cs

```diff
@@ -0,0 +1,177 @@
+﻿// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+using Mozilla.Glean.FFI;
+using System;
+using System.IO;
+
+namespace Mozilla.Glean
+{
+    /// <summary>
+    /// The Glean Gneral API.
+    /// </summary>
+    public class Glean
```

Comment:
Let's make this `sealed` also to prevent it from being inherited from.

Reply:
Good point!